### ISSUE TITLE
GUI : Design : FRUs Concurrently Maintainable

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#1521]
## SMEs
**BMC**: TBD
**RAS**: Justin Thaler

## Use Case
TBD

## InVision Prototype
TBD

## Design Issue (phosphor-webui)
TBD

## References/Resources
- [eBMC Feature Item: 2.2.5](https://ibm.box.com/s/j15ux3yfjycy4or4azbqyhqq11lbya0r)
- feature discovery folder: 
- user research notes*: 
- user research synthesis: 

* This folder is restricted in accordance with GDPR guidelines.

**Comment by nicoleconser** [comment]
TOD Battery = RTC Batter (time of day / real time clock)

**Comment by mzipse** [comment]
@derick-montague , just fyi.  I'm working to get some clarification on the concurrently maintainable device vs the hot pluggable devices.  We're also checking on differences between our platforms.  I'll keep you posted.

**Comment by derick-montague** [comment]
Concurrent Maintenance and what is Concurrent Maintainable for R / E.

https://w3-connections.ibm.com/wikis/home?lang=en-us#!/wiki/W5d74d2bb8660_4d6e_9bb6_08e4fc6ff375/page/OpenBMC%20Downstream%20Design%20Meeting%20Topics

**Comment by nicoleconser** [comment]
From [Gunnar](https://ibm-systems-power.slack.com/archives/GBVDZRCQK/p1595254273246600): https://w3-connections.ibm.com/wikis/home?lang=en-us#!/wiki/W5d74d2bb8660_4d6e_9bb6_08e4fc6ff375/page/OpenBMC%20Downstream%20Design%20Meeting%20Topics has some gold on Concurrent Maintenance and what is Concurrently Maintainable

**Comment by nicoleconser** [comment]
Note that this includes the closed story #2237.

**Comment by derick-montague** [comment]
Mini-workshops happening on 8/4/20 and 8/5/20, technical discussion let by Santosh.

The intent is to hold two workshops related to HW Concurrent Maintenance:
1. Part 1 (8/4/20 9am CST) - Address eBMC external interfaces to support Redfish, HMC and eBMC GUI
1. Part 2 (8/5/20 9am CST) - Address Host interfaces to eBMC

I will be attending these workshops and will create and add information to a discovery document.

**Comment by nicoleconser** [comment]
**8/7 sync-up notes:** 
- Need to find out what this page looks like in ASMI (Russ says this is on P8s only) 
- Unclear that this belongs on the Hardware Inventory / VPD page (Justin says yes, Russ says totally unrelated)

**Comment by priyanka-pillai97** [comment]
- TPM affects the security of host boot process. It is a setting which is set once and then not changed frequently.
- VTPM (Virtual TPM) is not related to host boot process. It affects the security of logical partitions. It is a setting which is set once and then not changed frequently.
Validated by Nick B (Hostboot) - Understanding is correct; TPM covers boot firmware, vTPM covers logical partitions. 

FYI @nicoleconser

**Comment by priyanka-pillai97** [comment]
Concurrently Maintainable FRUs - 
- On Rainier, the following will be concurrently maintainable: PCIe Adapters, NVMe Drives (this is driven by the host, not BMC GUI). 
- On Everest, in addition to those two, the TOD Battery, Base Op Panel, and LCD Op panel are concurrently maintainable.

Slack conversation - https://ibm-systems-power.slack.com/archives/C0Q6TQP5Z/p1610963741022400

**Comment by derick-montague** [comment]
The current state of the backend requires that we schedule time in @bradbishop design exchange meeting. @derick-montague will we responsible for setting up time in the meeting.

**Comment by derick-montague** [comment]
Possibly moving out of GA1. Will know soon. As a side note, Rainier and Everest are different and TOD is not concurrently maintainable on Rainier systems, so the availability of this in the GUI will need to be based on Redfish indicating if the FRU supports CM for a specific FRU.

**Comment by derick-montague** [comment]
[Slack conversation regarding Ops Panel and TOD Redfish](https://ibm-systems-power.slack.com/archives/C0Q6TQP5Z/p1616513682220100)

## TL;DR
Brad:
> Santosh and vishwa for panel/battery CM it sounds like we need a Redfish resource to call a method on to start the operation?

Gunnar:
> So we need the "ReadyForRemove" on assembly ?

**Comment by mzipse** [comment]
Work is complete.  Closing.
